src/main/kotlin/graphics/scenery/backends/vulkan/VulkanRenderer.kt

```diff
@@ -380,7 +382,7 @@ open class VulkanRenderer(hub: Hub,
     var fps = 0
         protected set
     protected var frames = 0
-    protected var totalFrames = 0L
+    var totalFrames = 0L
```

Comment:
Should be `private set`.

Reply:
Also, it should probably be moved to `Renderer`.

Reply:
Changed to `protected set` and moved to `Renderer`. `private set` does not compile